### ISSUE TITLE
Error while running with nohup

### ISSUE THREAD

**Issue by dm222** [websockets/wscat#118]
When run
`wscat -c "127.0.0.1:8000" -x 'text'`
Message is sent

If run
`nohup wscat -c "127.0.0.1:8000" -x 'text' &`
getting an error:
```
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: EBADF: bad file descriptor, read
```
Version wscat is 4.0.0
Lubuntu 18.04, kernel 5.3.0-40-generic

**Comment by andypayne** [comment]
Does running it in a subshell work for you?
`(nohup wscat -c "127.0.0.1:8000" -x 'text' &)`
or
`sh -c 'nohup wscat -c "127.0.0.1:8000" -x "text" &'`
or
`nohup sh -c 'bin/wscat -c "127.0.0.1:8000" -x "text" &'`

**Comment by haug1** [comment]
No, it does not work. Immediate exit. Alpine linux, please try to fix

**Comment by akshaykhairmode** [comment]
I am working on a CLI tool similar to wscat to support nohup commands.

You can check it out if you like. https://github.com/akshaykhairmode/wscli/releases/tag/v0.0.1
